pkg/util/k8sutil/backup.go

```diff
@@ -36,7 +36,7 @@ import (
 )
 
 const (
-	storageClassPrefix        = "etcd-backup"
+	StorageClassPrefix        = "etcd-backup"
```

Comment:
doc string on public field (i know other public fields are undocumented, but we need to improve this over time as the operator stabilizes)

thank you!